src/assets/wise5/authoringTool/components/translatable-input/translatable-input.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, ContentChild, ElementRef, Input, Signal, computed } from '@angular/core';
+import { TeacherProjectService } from '../../../services/teacherProjectService';
+import { CommonModule } from '@angular/common';
+import { MatInput, MatInputModule } from '@angular/material/input';
+import { FormsModule } from '@angular/forms';
+import { TranslateProjectService } from '../../../services/translateProjectService';
+import { MatLabel } from '@angular/material/form-field';
+
+@Component({
+  standalone: true,
+  selector: 'translatable-input',
+  imports: [CommonModule, FormsModule, MatInputModule],
+  styleUrls: ['./translatable-input.component.scss'],
+  templateUrl: './translatable-input.component.html'
+})
+export class TranslatableInputComponent {
+  @Input() content: object;
+  @Input() key: string;
+  @ContentChild(MatInput) input: MatInput;
```

Comment:
Maybe the input variable can be renamed to something like contentInput to be a little more clear particularly when reading the template file.